### PR TITLE
raspberry sensors

### PR DESCRIPTION
**Pull request by crcro**
added snmp extend to get raspberry sensors

### CODE REVIEW

File: snmp/raspberry.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+getFreqArm='measure_clock arm'
+getFreqCore='measure_clock core'
+getStatusH264='codec_enabled H264'
+getStatusMPG2='codec_enabled MPG2'
+getStatusWVC1='codec_enabled WVC1'
+getStatusMPG4='codec_enabled MPG4'
+getStatusMJPG='codec_enabled MJPG'
+getStatusWMV9='codec_enabled WMV9'
+
+sudo $picmd $getTemp | $pised 's|[^0-9.]||g'
```
Comment (murrant) [inline snmp/raspberry.sh]: Is sudo required?

Comment (crcro) [inline snmp/raspberry.sh]: unfortunately yes, because vcgencmd won't return any data if not run by root

Comment (murrant) [inline snmp/raspberry.sh]: If any extra configuration is needed to allow sudo to run, make sure you add it to the docs in you PR on the main repo.

Comment (crcro) [inline snmp/raspberry.sh]: i did that already on main repo with PR #4057

**Pull request merged**